Samples/AuthenticationFlows.swift

```diff
@@ -38,7 +38,7 @@ func signInWithWebUsingCustomConfiguration() async throws {
                                  redirectUri: redirectUrl)
     
     // Sign in using the above configuration
-    let token = try await auth.start(from: view.window)
+    let token = try await auth.signIn(from: view.window)
```

Comment:
How does this change the behavior?

Reply:
This is just sample code, but a long time ago when I renamed the `start` method to `signIn`, I forgot to update the sample.

Reply:
Ah got it. Thanks for clarifying!